identity/controller.go

```diff
@@ -0,0 +1,41 @@
+package identity
+
+import "github.com/mysterium/node/service_discovery/dto"
+
+type identityController struct {
```

Comment:
Naming. Usually name Controller stands for C in MVC paradigm and it is something facing user, handling its actions and rendering views. Maybe indentityController can be renamed to something like identityManager to avoid confusion with widely adopted terms.

Reply:
how about identityHandler?

Reply:
renamed to identityHandler